meetings/cpansec-minutes-2025-01-08.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: page
+toc: true
+meeting_time: 2025-01-08 16:00 UTC
+title: Minutes 2025-01-08
+---
+
+## Agenda & Meeting Details
+
+* 2025-01-08 16:00 UTC on #cpansec-discussion on Matrix
+
+## 15:30 UTC – Pre-meeting socializing
+
+*   Socializing & getting up to speed before the meeting starts properly
+*   Discuss organizing projects, swimlanes and issues (...)
+
+## 16:05 UTC – Meeting start
+
+### Welcome
+
+*   Meeting chair: @sjn
+*   Meeting secretary: @tux & @stigtsp
+
+### Attendees, absents & regrets
+
+*   Attendees
+    * @sjn, @robrwo. @tux, @leont, @timlegge, @stigtsp
+*   Regrets
+    * …
+
+### Approve previous meeting minutes
+*   Previous meeting minutes was approved by @timlegge and @robrwo, and merged by @sjn
+
+
+### Quick summary of current work (Grouped by project)
+, 
+1.  [CPAN Metadata & Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1)
+    *   …
+2.  [CPAN Privacy and Compliance](https://github.com/orgs/CPAN-Security/projects/9)
+    *   …
+3.  [CPAN Provenance & Supply Chain Security](https://github.com/orgs/CPAN-Security/projects/3)
+    *   …
+4.  [CPAN Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+    *   New and existing documentation being discussed in relation to open subjects
+5.  [CPAN Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+    *   …
+6.  [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+7.  [CPAN Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+    *   …
+8.  [CPAN Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+    *   …
+9.  [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+    *   …
+10. [CPAN Secure by Default](https://github.com/orgs/CPAN-Security/projects/15)
+    *   …
+11. [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+
+### TLS/HTTPS/CSPRNG/DSA in core
+- @leont - Worked on BearSSL & OpenSSL
+  - cleaning up CORE is required but tedious
```

Comment:
It's not core that needs cleaning up, it's `Crypt::OpenSSL`/`Net::SSLeay` (and lesser extend `IO::Socket::TCP`/`IO::Socket::SSL`